content/src/content/jcr_root/apps/core/wcm/components/form/options/v2/options/amp.html

```diff
@@ -0,0 +1,39 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2016 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+<sly data-sly-use.options="com.adobe.cq.wcm.core.components.models.form.Options"
+     data-sly-use.template="core/wcm/components/commons/v1/templates.html"
+     data-sly-test.isNotEmpty="${options.items.size}">
+    <fieldset class="cmp-form-options cmp-form-options--${options.type.value}">
+        <sly data-sly-test="${options.type.value == 'checkbox' || options.type.value == 'radio'}">
+            <legend class="cmp-form-options__legend">${options.title}</legend>
+            <label class="cmp-form-options__field-label" data-sly-repeat.optionItem="${options.items}">
+                <input class="cmp-form-options__field cmp-form-options__field--${options.type.value}" checked="${optionItem.selected}" type="${options.type.value}" name="${options.name}"
+                       value="${optionItem.value}" disabled="${optionItem.disabled}"/>
```

Comment:
AMP takes issue with either `checked` or `required` being the last attribute of an input. It reads `checked/` rather than `checked` due to the closing `/` of the input. So, the only update here between AMP and non-AMP is that the `checked` attribute moved higher in the input so that the issue will not occur on AMP pages.

Reply:
Sounds good. Should not be a problem.